scope.go

```diff
@@ -23,12 +23,14 @@ package tally
 import (
 	"fmt"
 	"sync"
+	"time"
 
 	"github.com/facebookgo/clock"
 )
 
 type scope interface {
```

Comment:
"scope" is included in the "Scope" interface. It contains the non-exported methods